File: iOSDemo/iOSDemo/Others/Controller/sub/BaseOnlyBarVC.m
//
//  BaseOnlyBarVC.m
//  AnyThinkSDKDemo
//
//  Created by TopOn技术支持 on 2025/7/3.
//

#import "BaseOnlyBarVC.h"

@interface BaseOnlyBarVC ()

@end

@implementation BaseOnlyBarVC

#pragma mark - Demo Needed (No need to integrate)
- (void)viewDidLoad {
    [super viewDidLoad];
     
    [self setupDemoUI];
}

- (void)setupDemoUI {
    [self addNormalBarWithTitle:self.title];
}

@end

File: SwiftDemo/SwiftDemo/Others/Controller/sub/BaseOnlyBarVC.swift
//
//  BaseOnlyBarVC.swift
//  SwiftDemo
//
//  Created by ltz on 2025/8/20.
//

import UIKit

class BaseOnlyBarVC: BaseVC {
    
    // MARK: - Demo Needed (No need to integrate)
    override func viewDidLoad() {
        super.viewDidLoad()
        
        setupDemoUI()
    }
    
    func setupDemoUI() {
        addNormalBar(withTitle: title)
    }
}
